incubator/hnc/Makefile

```diff
@@ -155,7 +156,11 @@ manifests: controller-gen
 
 # Run go fmt against code
 fmt:
-	go fmt ${DIRS}
+	gofmt -w ${GOFMT_DIRS}
```

Comment:
Why the switch from `go fmt` to `gofmt`?

Reply:
In short, `go fmt` runs `gofmt -l -w` on the packages specified by the arguments.

As we include `check-fmt` make target using `gofmt`, I thought it was a good idea to use the same command.

Reply:
Sorry for not understanding. Why do we _not_ want to use the `-l` flag where we were using it before? And why can't we use `-l` in `check-fmt:`? Per the docs:

```
    -l
    	Do not print reformatted sources to standard output.
    	If a file's formatting is different from gofmt's, print its name
    	to standard output.
```

How about this: in the CI script, we just run `make fmt` and see if it prints anything to stdout? If nothing happens, then we know the files were all in a good state. We don't need any modifications to the makefile at all?

(Context: I'm trying to avoid having two vars with duplicated data that need to be kept in sync, as these directories do change fairly frequently)

Reply:
Ye, it was my fault. I missed the `-l` in the `gofmt -w` command. Solved.
I don't like to implement this kind of logic in the CI script, `make` targets are cleaner IMHO.

Also, we solved the issue of having two variables. Thanks to @rjbez17 who gave us the key!